core/store/orm/orm_test.go

```diff
@@ -1357,3 +1357,29 @@ func TestJobs_ScopedInitiator(t *testing.T) {
 	expectation := []string{fmJob.ID.String(), twoInitrJob.ID.String()}
 	assert.ElementsMatch(t, expectation, actual)
 }
+
+func TestJobs_MoreThanBatchWithArchives(t *testing.T) {
+	store, cleanup := cltest.NewStore(t)
+	defer cleanup()
+
+	archivedJob := cltest.NewJobWithFluxMonitorInitiator()
+	archivedJob.DeletedAt = cltest.NullableTime(time.Now())
+	require.NoError(t, store.CreateJob(&archivedJob))
+
+	jobs := []models.JobSpec{}
+	jobNumber := 202
```

Comment:
If a constant is used for the Batch number, it can be referenced here for the test.